Tools/Scripts/hooks/pre-push

```diff
@@ -292,11 +292,9 @@ def main(name, remote):
         if not match:
             print_error("'{}' does not conform to a recognized target remote".format(remote))
             return 255
-        name = remote_to_name.get('{}:{}'.format(match.group('host'), match.group('path')), None)
-        if not name:
+        name = remote_to_name.get('{}:{}'.format(match.group('host'), match.group('path')), remote)
+        if name not in remote_to_name.values():
```

Comment:
```suggestion
        if name not in name_to_remote:
```

No point in doing the relatively slow thing here. But r+'d if you fix that.

Reply:
We need to search `values()` not `keys()`, though. I could change this to:
```
name = remote_to_name.get('{}:{}'.format(match.group('host'), match.group('path')), None)
if not name:
    name = remote
    ...
```
is that more clear maybe?

Reply:
As mentioned in DMs: note `remote_to_name` v. `name_to_remote`; it's a different dict.